tests/testthat/test-paginate_listing.R

```diff
@@ -170,19 +174,15 @@ testthat::test_that("pagination: lpp and cpp correctly computed for margins", {
 })
 
 testthat::test_that("pagination works with col wrapping", {
-  lsting <- h_lsting_adae(disp_cols = c("USUBJID", "AESOC", "RACE"))
+## This test was predicated and enforcing the extra colgap bug in https://github.com/insightsengineering/formatters/issues/249
```

Comment:
I am not sure we can comment code in tests. Probably better to keep it and then change it in the colgap PR

Reply:
#249

Reply:
It doesn't pass which is why it is commented out, I had to fix the referenced formatters bug in order to get everything working. We can't keep it unless we specifically add the bug back into my PR which would be non-trivial (and silly to do)

Reply:
I can delete it entirely instead of leaving it there commented out if that is preferable

Reply:
Could you change it to test that the bug is not present? otherwise we can remove

Reply:
There is a test in the formatters PR for this, is that sufficient?